PyDSS/ResultData.py

```diff
@@ -144,7 +132,7 @@ def UpdateResults(self):
 
     def ExportResults(self, fileprefix=""):
         self._export_indices()
-        self._export_event_log()
+        #self._export_event_log()
```

Comment:
Why did you comment this out?

Reply:
This producing an error for some wierd reason when I was running test. I comment for the time being. Will uncomment it. The issue was in the dss file itself.

Reply:
fixed